src/rules/quotemarkRule.ts

```diff
@@ -39,16 +39,20 @@ export class Rule extends Lint.Rules.AbstractRule {
 
 class QuoteWalker extends Lint.RuleWalker {
     private quoteMark: QuoteMark;
```

Comment:
let's avoid having to handle the error case where the `quoteMarkString` parsed from options is invalid and initialize this value to a default:

```ts
private quoteMark = QuoteMark.DOUBLE_QUOTES;
```